src/Controller/User/SignupController.php

```diff
@@ -134,7 +134,7 @@ public function signupAction(Request $request)
                 if ('dev' === $this->kernelEnv) {
                     $form->addError(new FormError(get_class($e).': '.$e->getMessage()));
                 } else {
-                    $form->addError(new FormError('Sorry, an error occurred while creating your account.'));
+                    $form->addError(new FormError('Sorry, Passwords must be at least 8 characters long.'));
```

Comment:
This is not the only error that could occur, should check for the specific error.

AddUserCommand should have the assertion updated for the new requirement as well.